tests/unit_tests/commands/dataset/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@betodealmeida I gather this file is now obsolete.

Reply:
Ah, good point.